src/api/userApi.js

```diff
@@ -0,0 +1,15 @@
+import { handleError, handleResponse } from './apiHelpers';
+
+// we should use .env variables for constants and keys
+const url = 'http://localhost:3002/friends/';
```

Comment:
Due to plans, launching mappypals on Heroku. This url is not ideal. You should add something along with this.
` const url = process.env.URL || 'http://localhost:3001/';`

Reply:
Totally agree!

Reply:
I was wondering if can do this:
- create a file .env.development
- define REACT_APP_URL='http://localhost:3002/'
and then in the code
```javascript
const url = process.env.REACT_APP_URL + 'friends'
```

Reply:
I didn't realize that I added this as a single comment instead of an as a whole review. Anyways full review is now live @sandrohp88 Yes you can do that. There is a task for that. But you can implement it in this PR. https://trello.com/c/9gHrc6vx/111-adding-env-file-to-front-end